api/src/client/DynamicsBusinessIdsClient.ts

```diff
@@ -0,0 +1,36 @@
+import { LogWriterType } from "@libs/logWriter";
+import axios, { AxiosError } from "axios";
+import { BusinessIdClient, BusinessIdResponse } from "../domain/types";
+
+export const DynamicsBusinessIdsClient = (logWriter: LogWriterType, orgUrl: string): BusinessIdClient => {
+  const getBusinessId = async (accessToken: string, nameToSearch: string): Promise<string[]> => {
```

Comment:
[sand] I might rename this `getBusinessIds` because it returns an array of strings. Or something even more descriptive, because it's based on a name search? `getMatchingBusinessIds` or something?

Reply:
I changed it to `getBusinessIds` because `getMatchingBusinessIds` might be confusing as we use the `matching` prefix later on in the file to describe the business id after processing.